teams/vsp/teams/operations/onboarding.md

```diff
@@ -1,18 +1,18 @@
-# Onboarding Operations Team Members
+# Onboarding Infrastructure Team Members
 
 This document is meant as a way to assist a member of the VSP Operations team to get started. It starts with higher level organization of both technology and people, provides some information on how the team communicates, and adds some jumping off points into more details of how the VSP works.
 
 ## Kicking things off
 
 A great introduction to start is the [TechnicalArchitectureOverview]() page.
 
-There's also some high level documentation about how the [VA Enterprise Cloud transit VPC is constructed](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/aws-dx-transit-spoke-architecture/architecture.md) and how the platform fits within that ecosystem. This document is from the initial implementation phase, but from a high level a lot of the main constructs are still there. We also have a video of an overview of the [VA network overall](https://dsva.slack.com/archives/C0S6H6BQU/p1569526014067200) that is super helpful.
+There's also some high level documentation about how the [VA Enterprise Cloud transit VPC is constructed](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/aws-dx-transit-spoke-architecture/architecture.md) and how the platform fits within that ecosystem. This document is from the initial implementation phase, but from a high level a lot of the main constructs are still there. We also have a video of an overview of the [VA network overall](https://dsva.slack.com/archives/C0S6H6BQU/p1569526014067200) (75 min.) that is super helpful.
 
 Speaking of VAEC and how things fit together, some introductions to the organization [here](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/VA-IT-Organization-and-Contacts.md).
 
-There are currently 3 environments for the applications we deploy: dev, staging, and prod. A fourth is coming called sandbox. We also deploy into a utility environment for tools that don't fit cleanly that we only have one of (Jenkins, Grafana, etc).
+There are currently four environments for the applications we deploy: dev, staging, prod and sandbox. We also deploy into a utility environment for tools that don't fit cleanly that we only have one of (Jenkins, Grafana, etc).
 
-On the application side there's a (somewhat older, but still relevant) video of [how endpoints get added to the API](https://www.youtube.com/watch?v=V_i8JLXk5rg&feature=youtu.be) and some details about how the app works at that level.
+This video doesn't exist on youtube anymore :/ Is there something else we can link here?
```

Comment:
Question here.

Reply:
Not sure on this one

Reply:
Perhaps https://depo-platform-documentation.scrollhelp.site/developer-docs/Adding-a-new-External-Service-Integration.1884586162.html coupled with https://vfs.atlassian.net/wiki/spaces/OT/pages/1474594279/Forward+Proxy